Develop an SQL schema for SOPs

```diff
@@ -0,0 +1,4351 @@
+--
+
+ALTER SEQUENCE public."fittedMeasuredFluxResiduals_id _seq" OWNED BY public."fittedMeasuredFluxResiduals"."id ";
+
+
+--
+-- TOC entry 231 (class 1259 OID 17871)
+-- Name: fittedMeasuredFluxes; Type: TABLE; Schema: public; Owner: postgres
+--
+
+CREATE TABLE public."fittedMeasuredFluxes" (
+    id integer NOT NULL,
+    simulation_id text,
+    "simulation_dateAndTime" timestamp(6) without time zone,
+    experiment_id text,
+    sample_name_abbreviation text,
+    " rxn_id" text,
+    fitted_sres double precision,
+    used boolean,
+    comment text
+);
+
+
+ALTER TABLE public."fittedMeasuredFluxes" OWNER TO postgres;
+
+--
+-- TOC entry 252 (class 1259 OID 18030)
+-- Name: fittedMeasuredFluxes_id_seq; Type: SEQUENCE; Schema: public; Owner: postgres
+--
+
+CREATE SEQUENCE public."fittedMeasuredFluxes_id_seq"
+    AS integer
+    START WITH 1
+    INCREMENT BY 1
+    NO MINVALUE
+    NO MAXVALUE
+    CACHE 1;
+
+
+ALTER TABLE public."fittedMeasuredFluxes_id_seq" OWNER TO postgres;
+
+--
+-- TOC entry 3149 (class 0 OID 0)
+-- Dependencies: 252
+-- Name: fittedMeasuredFluxes_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: postgres
+--
+
+ALTER SEQUENCE public."fittedMeasuredFluxes_id_seq" OWNED BY public."fittedMeasuredFluxes".id;
+
+
+--
+-- TOC entry 232 (class 1259 OID 17874)
+-- Name: fittedMeasuredFragmentResiduals; Type: TABLE; Schema: public; Owner: postgres
+--
+
+CREATE TABLE public."fittedMeasuredFragmentResiduals" (
+    " id" integer NOT NULL,
+    simulation_id text,
+    "simulation_dateAndTime" timestamp(6) without time zone,
+    experiment_id text,
+    "sample_name_abbreviation " text,
+    " time_point" text,
+    "fragment_id " text,
+    fragment_mass integer,
+    res_data double precision,
+    "res_esens " double precision,
+    res_fit double precision,
+    res_msens double precision,
+    res_peak text,
+    res_stdev double precision,
+    res_val double precision,
+    used boolean,
+    comment text
+);
+
+
+ALTER TABLE public."fittedMeasuredFragmentResiduals" OWNER TO postgres;
+
+--
+-- TOC entry 253 (class 1259 OID 18041)
+-- Name: fittedMeasuredFragmentResiduals_ id_seq; Type: SEQUENCE; Schema: public; Owner: postgres
+--
+
+CREATE SEQUENCE public."fittedMeasuredFragmentResiduals_ id_seq"
+    AS integer
+    START WITH 1
+    INCREMENT BY 1
+    NO MINVALUE
+    NO MAXVALUE
+    CACHE 1;
+
+
+ALTER TABLE public."fittedMeasuredFragmentResiduals_ id_seq" OWNER TO postgres;
+
+--
+-- TOC entry 3150 (class 0 OID 0)
+-- Dependencies: 253
+-- Name: fittedMeasuredFragmentResiduals_ id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: postgres
+--
+
+ALTER SEQUENCE public."fittedMeasuredFragmentResiduals_ id_seq" OWNED BY public."fittedMeasuredFragmentResiduals"." id";
+
+
+--
+-- TOC entry 233 (class 1259 OID 17877)
+-- Name: fittedMeasuredFragments; Type: TABLE; Schema: public; Owner: postgres
+--
+
+CREATE TABLE public."fittedMeasuredFragments" (
+    "id " integer NOT NULL,
```

Comment:
Please remove ALL of the extra white spaces here and below.  I will stop pointing them out individually from now on.

Reply:
ok